final release/format_data.py

```diff
@@ -0,0 +1,208 @@
+
+# coding: utf-8
+
+# In[45]:
+
+
+import numpy as np
+import pickle
+import matplotlib.pyplot as plt
+import h5py
+from skimage.transform import resize
+from astropy.io import ascii
+#import ask_first
+from sklearn.utils import shuffle
+from keras.utils import np_utils
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+
+import sys, os
+
+
+# In[46]:
+
+def get_data(first_path, second_path, dim = 256, seed = 0, conf_matrix = True, flip_train_test = False):
+    # Loading data
+    data = h5py.File(first_path, 'r')
+
+    images = np.asarray(data['images'])
+    labels = np.asarray(data['labels'])
+
+    # Opening Catalogues
+    asu = ascii.read(second_path).to_pandas()
+
+    # Opening Entire First Catalogue with ask_first script (Only for reference)
+    #first = ask_first.read_catalogue("catalog_14dec17.bin")
+
+
+    # Splitting classes
+    fri_images = images[np.where(labels == False)]
+    frii_images = images[np.where(labels == True)]
+
+    # Extracting data 
+    fri_data = data['fri_data']
+    frii_data = data['frii_data']
+
+    # Converting coordinates into a Astropy Skycoord readable format
+    frii_c = np.hstack((np.asarray(list(frii_data['_RA'])).reshape(-1, 1), np.asarray(list(frii_data['_DE'])).reshape(-1, 1)))
```

Comment:
This line is too long. Break it up or break it down.

Reply:
done